Build breaks with clang-19: no member named 'get' in 'ParameterProxy<T>'

```
In file included from src/solvers/twostep_time_solver.C:26:
In file included from ./include/libmesh/parameter_vector.h:26:
./include/libmesh/parameter_accessor.h:115:95: error: no member named 'get' in 'ParameterProxy<T>'
  115 |   ParameterProxy & operator = (const ParameterProxy<T> & new_value) { _accessor.set(new_value.get()); }
      |                                                                                     ~~~~~~~~~ ^
```

This patch removing unused methods should help:
```
--- include/systems/parameter_accessor.h.orig   2024-11-04 03:11:10 UTC
+++ include/systems/parameter_accessor.h
@@ -112,13 +112,14 @@ class ParameterProxy (public)
   /**
    * Setter: change the value of the parameter we access.
    */
+#if 0
   ParameterProxy & operator = (const ParameterProxy<T> & new_value) { _accessor.set(new_value.get()); }
 
   /**
    * Setter: change the value of the parameter we access.
    */
   ParameterProxy & operator = (const ConstParameterProxy<T> & new_value) { _accessor.set(new_value.get()); return *this; }
-
+#endif
   /**
    * Setter: change the value of the parameter we access.
    */
```

Version: 1.7.5
FreeBSD 14.1

--- a/include/systems/parameter_accessor.h
+++ b/include/systems/parameter_accessor.h
@@ -112,7 +112,7 @@ public:
   /**
    * Setter: change the value of the parameter we access.
    */
-  ParameterProxy & operator = (const ParameterProxy<T> & new_value) { _accessor.set(new_value.get()); }
+  ParameterProxy & operator = (const ParameterProxy<T> & new_value) { _accessor.set(new_value._accessor.get()); return *this; }
 
   /**
    * Setter: change the value of the parameter we access.
@@ -143,6 +143,11 @@ public:
    * Getter: get the value of the parameter we access.
    */
   operator T () const { return _accessor.get(); }
+
+  /**
+   * Getter: get the value of the parameter we access.
+   */
+  T get() const { return _accessor.get(); }
 
 #ifdef LIBMESH_DEFAULT_QUADRUPLE_PRECISION
   operator boost::multiprecision::backends::float128_backend () const { return _accessor.get().backend(); }

